addons/account/models/account_tax.py

```diff
@@ -288,10 +288,10 @@ def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, p
             return base_amount - (base_amount / (1 + self.amount / 100))
         # base / (1 - tax_amount) = new_base
         if self.amount_type == 'division' and not price_include:
-            return base_amount / (1 - self.amount / 100) - base_amount if (1 - self.amount / 100) else 0.0
+            return base_amount - (base_amount / (1 + self.amount / 100))
```

Comment:
@smetl this one doesn't seem to work properly based on the expected result.
Odoo functional documentation with the expected result when having this Percentage of Price tax included - and not selecting the "Tax Include" boolean-  with sales price 1000: 
https://www.odoo.com/documentation/16.0/applications/finance/accounting/taxes.html#tax-computation
<img width="1070" alt="image" src="https://github.com/odoo/odoo/assets/45669815/bb9bcbee-d4a9-4b61-8a84-1d3eccd18279">

Here is what I am getting in runbot (
<img width="1159" alt="image" src="https://github.com/odoo/odoo/assets/45669815/6de21b53-7a0d-42ba-aa67-df8bb66f444e">
)when testing with this configuration:
<img width="1332" alt="image" src="https://github.com/odoo/odoo/assets/45669815/11992a48-b405-4599-98b9-cda4437a1942">
<img width="1159" alt="image" src="https://github.com/odoo/odoo/assets/45669815/6e1d9bab-953f-4c38-adfc-4bcce412a925">

The Brazilian use case is working as expected (when config is Percentage of Price Tax Included + Included in Price sleected), but it seems that this one is broken (based on the expected result)

cc' @tsb-odoo @chklop

Reply:
@smetl Based on a testing performend on a runbot DB without this FIX, it was working as expected.
So: is there an option to keep this formula as it was [return base_amount / (1 - self.amount / 100) - base_amount if (1 - self.amount / 100) else 0.0] but only modifying the one below?